java/java.hints/src/org/netbeans/modules/java/hints/jdk/Bundle.properties

```diff
@@ -111,3 +111,12 @@ DESC_AnnoProcessor_ObsoleteSupportedSource=If the project's source level is grea
     another useless warning during compilation. <p/>\
     It is recommended to return at least the current project's source level. \
     For future compatibility, consider to return <b>SourceVersion.latest()</b>; most Processors are not affected by future language changes.
+CheckRegexTopComponent.regexLabel.text=Regular Expression:
```

Comment:
Maybe we could annotate shortcut keys for both JTextAreas and JCheckBox components with & character?

Reply:
Thanks. Have made the changes.

Reply:
Shortcuts work great now!